PUBLISHING.md

```diff
@@ -1,13 +1,30 @@
 # Publishing Instructions
 
-To update the version number, edit the version in `components/package.json`. 
+## Automatic Publishing
 
-To publish a new package through NPM, run the following commands from the root folder: 
+This package is published to NPM automatically by CircleCI when code is merged into the `dev` or `master` branches. To publish a new version, simply update the version in `package.json` and merge your code into the appropriate branch.
+- The `dev` branch will publish versions marked as `alpha` or `beta`.
+- The `master` branch will publish any version (`alpha`, `beta`, or `latest`).
+  In both cases, the code will only be published if the version number differs from the current version published under the respective dist tag.
 
+## Manually Publishing
+
+If you need to publish a package manually, you can run the following commands from the root folder:
+
+```sh
+yarn build 
+cd dist
+yarn publish:package
```

Comment:
Same comment here about moving the cd into the publish command

Reply:
This one should work as-is because the scripts section isn't removed as part of the build.  I can make all 3 component libs have the same top-level publish command for consistency though.

Reply:
That way everything will look the same in the CI config file and the publishing doc

Reply:
Ok RN and React libraries match now.